### DISCUSSION TITLE
Add a feature on Katana - No permission to push

### DISCUSSION THREAD

**Discussion by alban-stourbe-wmx** [projectdiscovery/katana#920] [Q&A] [answered]
Hello everyone, 
I've created a new feature to add textarea and select detection in autofill form. I've created my own branch and added a few tests. Nevertheless, I can't seem to push it to create a pull request. 

<img width="795" alt="Capture d’écran 2024-06-06 à 16 44 14" src="https://github.com/projectdiscovery/katana/assets/159776828/7135b955-1433-4536-937b-c3aa28b86deb">

Is it possible to add features even if you're not part of projectdiscovey?

Thanks all :)

**Comment by Mzack9999** [comment, marked as answer]
@alban-stourbe-wmx You need to create a fork of the repository at https://github.com/projectdiscovery/katana/fork, implement your own changes on the `dev` branch then open a pull request upstream.

Meanwhile, you can join our [discord server](https://discord.com/invite/projectdiscovery). It's a great place to connect with fellow contributors and stay updated with the latest developments.

Additionally, if you could kindly DM me on Discord (mzack9999), we would like to express our gratitude by sending you some swag for your outstanding contribution. Thank you once again.

**Reply by alban-stourbe-wmx** [reply]
@Mzack9999 Thanks a lot. 
I already join the Discord few weeks ago ! :) 
Of course, i am going to DM right now !
